src/functions/getConfiguration/framework/handler.ts

```diff
@@ -52,25 +51,12 @@ export async function handler(event: APIGatewayProxyEvent): Promise<Response> {
 
   const scope: Scope = event.pathParameters.scope as Scope;
   info('Returning configuration for ', scope);
-  customMetric('ConfigurationReturned', 'Number of times the configuration has been returned to a user');
 
-  const config: RemoteConfig = await buildConfig(staffNumber, examinerRole, formattedAppVersion);
+  const config: RemoteConfig = await buildConfig(staffNumber, examinerRole);
   const configClone = cloneDeep(config);
-  // delete team journals url & driver property if not coming from app version 4 or above
-  if (!isAppVersionEligibleForTeamJournal(formattedAppVersion)) {
-    const { driver, ...des3Config } = configClone;
-    delete des3Config.journal.teamJournalUrl;
-    return createResponse(des3Config);
-  }
-  // delete
-  if (!isAppVersionEligibleForDriverVehicle(formattedAppVersion)) {
-    const { driver, vehicle, ...configTrimmed } = configClone;
-    return createResponse(configTrimmed);
-  }
-  if (!isAppVersionEligibleForRefDataTestCentre(formattedAppVersion)) {
```

Comment:
If we need checks like this for pilot releases etc in future, then we can use something like this for example
```
if (!isEligibleFor(formattedAppVersion, '4.8.0.5)) {
   const { thingToExclude, ...configTrimmed } = configClone;
   return createResponse(configTrimmed);
}
```